readthedocs/templates/projects/project_import.html

```diff
@@ -93,41 +93,12 @@ <h1>{% trans "Import a Repository" %}</h1>
                       {% endblocktrans %}
                     </p>
 
-                    <div class="project-import-providers">
-                      <form
-                          method="get"
-                          action="{% provider_login_url "github" %}"
-                          class="import-connect import-connect-github">
-                        <button>
-                          {% trans "Connect to GitHub" %}
-                        </button>
-                        <input type="hidden" name="process" value="connect">
-                        <input type="hidden" name="next" value="/dashboard/import/">
-                      </form>
-
-                      <form
-                          method="get"
-                          action="{% provider_login_url "gitlab" %}"
-                          class="import-connect import-connect-gitlab">
-                        <button>
-                          {% trans "Connect to GitLab" %}
-                        </button>
-                        <input type="hidden" name="process" value="connect">
-                        <input type="hidden" name="next" value="/dashboard/import/">
-                      </form>
 
-                      <form
-                          method="get"
-                          action="{% provider_login_url "bitbucket_oauth2" %}"
-                          class="import-connect import-connect-bitbucket">
-                        <button>
-                          {% trans "Connect to Bitbucket" %}
-                        </button>
-                        <input type="hidden" name="process" value="connect">
-                        <input type="hidden" name="next" value="/dashboard/import/">
-                      </form>
+                    <div class="project-import-providers">
+                      <ul class="socialaccount_providers">
+                        {% include "socialaccount/snippets/provider_list.html" with process="connect" %}
```

Comment:
I don't see this file in the commit, did it already exist?

Reply:
Oh I see it below -- is it also used somewhere else?

Reply:
Yes, it is used in the `/accounts/social/connections/` page, under the admin settings.

Reply:
We do lose the `next` parameter here, correct? This will likely just redirect users back to the connection page. Is redirecting to the import page important here?

Reply:
Good catch! I didn't realize this.

Yes, I think it's important since you are at the import page. I'd like to connect my account and import the my project immediately. I'm proposing a fix for this using the template tag: https://github.com/pennersr/django-allauth/blob/cfc5112bf474af2cbb16e4b4a36708e8949a2105/allauth/socialaccount/templatetags/socialaccount.py#L45